Simplify Versioning

Jafaral opened the pull request:
Summary:
   The version is Major.Minor.Bugfix,  Bugfix will only show up on released branched if we do backport bug fixes to those branches. In most cases the User will only see Major.Minor, especially if they build from master. 

Once we release, the version number must be incremented

All branches are treated the same, we can still add (pre-release). Although the git extra info is an indication of pre-release.

Jafaral committed:
Simplify Versioning

Summary:
   The version is Major.Minor.Bugfix,  Bugfix will only show up on released branched if we do backport bug fixes to those branches. In most cases the User will only see Major.Minor, especially if they build from master. 

Once we release, the version number must be incremented

All branches are treated the same, we can still add (pre-release). Although the git extra info is an indication of pre-release.

Jafaral commented:
I did the edit directly on github, I don't know if I broke some syntax somewhere!

Don-Ward reviewed (commented):
The document no longer builds.  I'll incorporate your changes (and fix the build error) into my Autoversion branch, along with some suggested modifications.

When we're happy with that document, I suggest we drop this PR and make a PR from AutoVersion.

Don-Ward committed:
move (and rename) Unicon release document.

Don-Ward committed:
Fix build error; merge mods into original doc.

The document has been renamed and moved out of the top level directory.
Some typos have been corrected and the document now contains both
of the alternative labelling proposals.
